docs/performance/reduce-reconciliation.md

```diff
@@ -82,7 +82,7 @@ Now, if we add a new item to the end of the `Inventory` or change something abou
 
 Another performance improvement we can make is to use stable, unique keys to refer to our child elements.
 
-When the list that we pass into the `Inventory` component changes, Roact reconciles our Roblox UI by adjusting the properties of each primitive according to the new list of elements.
+When the list that we pass into the `Inventory` component changes, Roact reconciles our Roblox UI by adjusting the properties of each Roblox Instance according to the new list of elements.
```

Comment:
Let's drop the verb 'reconcile' here and just say 'update'. We should probably do that across the board!

Reply:
Any thoughts on the page name and some of our other references in here? `Roact.update` is better than `Roact.reconcile`, but I do still think that reconciliation is helpful as a term because it seems a little more specific.

Agreed that "update" is better here, though. I'll change it.

Reply:
Not sure. Reconciliation is probably fine!